src/app/search/search-list/search-list.component.spec.ts

```diff
@@ -6,16 +6,27 @@ import {ApplicableLocationObject, Location} from 'src/app/model/search-criteria'
 import { By } from '@angular/platform-browser';
 import { SearchDataService } from '../../services/search-data.serivce';
 import { SearchDataServiceMock } from '../../global-search/global-search.component.spec';
+import {environment} from '../../../environments/environment';
+import { PlacesMockService } from 'src/app/mock-services/places-mock/places-mock-service';
 
 describe ('SearchListComponent', ()=>{
     let component: SearchListComponent;
     let fixture: ComponentFixture<SearchListComponent>
-    let locationData ={location: [{}], position:{coords: {latitude: 3,longitude: 4}}};
+    let locationData ={location: [{ name: 'Chandigarh',details:{distance:{text:"500km"}}},{ name: 'Chandigarh',details: {distance:{text:"50km"}}}], position:{coords: {latitude: 3,longitude: 4}}};
```

Comment:
I think we have location mock data already defined. Please check if we can use that.

Reply:
Done